terms.html

```diff
@@ -133,10 +123,12 @@
 information to share.
   </dd>
   <dt><dfn data-lt="subjects|subject's|subject(s)">subject</dfn></dt>
-The <a>resource</a> about which a <a>claim</a> is made.
+The person or thing about which a <a>claim</a> is made. 
+This can not only be documents, people, organizations, physical objects, abstract concepts, etc.,
+but also <a>issuers</a>, <a>holders</a>, and <a>verifiers</a>.
```

Comment:
```suggestion
This can be documents, people, organizations, physical objects, abstract concepts, etc.,
as well as <a>issuers</a>, <a>holders</a>, and <a>verifiers</a>.
```